app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="layout_param">0dp</dimen>
+    <dimen name="margin_header_content">16dp</dimen>
+    <dimen name="text_size_settings">34sp</dimen>
+
+    <dimen name="text_size_logout_btn">14sp</dimen>
+    <dimen name="radius_logout_btn">8dp</dimen>
+    <dimen name="width_logout_btn">120dp</dimen>
+    <dimen name="height_logout_btn">40dp</dimen>
+    <dimen name="stroke_width_logout_btn">2dp</dimen>
+
+    <dimen name="width_img">114dp</dimen>
+    <dimen name="height_img">114dp</dimen>
+    <dimen name="margin_top_img">100dp</dimen>
+
+    <dimen name="margin_top_name">16dp</dimen>
+    <dimen name="text_size_name">24sp</dimen>
+
+    <dimen name="margin_top_profession">8dp</dimen>
+    <dimen name="text_size_profession">20sp</dimen>
+
+    <dimen name="margin_top_address">16dp</dimen>
+    <dimen name="margin_bottom_address">24dp</dimen>
+    <dimen name="text_size_address">14sp</dimen>
+
+    <dimen name="my_profile_down_height">0dp</dimen>
+
+    <dimen name="icon_margin_side">60dp</dimen>
+    <dimen name="icon_margin_top">80dp</dimen>
+
+    <dimen name="width_edit_btn">320dp</dimen>
+    <dimen name="height_edit_btn">60dp</dimen>
+    <dimen name="margin_edit_btn">16dp</dimen>
+    <dimen name="margin_bottom_edit_btn">16dp</dimen>
+    <dimen name="text_size_edit_btn">16sp</dimen>
+    <dimen name="radius_edit_btn">8dp</dimen>
+    <dimen name="stroke_width_edit_btn">2dp</dimen>
+
+    <dimen name="width_contacts_btn">320dp</dimen>
```

Comment:
every dimen that is greater than 100dp could possibly not fit on small screens, thus it is advisable to come up with some solutions to place the view relative to a screen's width, like using GuideLines or ConstraintLayout's biases

Reply:
I have fixed it